app/styles/core-v2/_variables.less

```diff
@@ -13,6 +13,19 @@
 
   --topbar-height: 56px;
 
+  --sidebar-width: 60px;
+  --sidebar-bg-color: var(--color-primary-500);
+  --sidebar-font-color: var(--color-white);
+  --sidebar-inner-border-color: var(--color-primary-400);
+  --sidebar-outer-border-color: var(--color-primary-500);
+  --sidebar-item-hover-bg-color: var(--color-primary-600);
+  --sidebar-item-pressed-bg-color: var(--color-primary-400);
+  --sidebar-item-pressed-box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.06), 0px 1px 3px rgba(0, 0, 0, 0.1),
```

Comment:
this is fully custom and can't match any of our existing box-shadows right ? :)

Reply:
Don't match any variable, the closest one is `box-shadow-xs`